common/reactive/src/main/java/io/helidon/common/reactive/Single.java

```diff
@@ -793,4 +803,17 @@ default T get(long timeout, TimeUnit unit) throws InterruptedException, Executio
 
     @Override
     CompletionAwaitable<T> exceptionally(Function<Throwable, ? extends T> fn);
+
+    /**
+     * Returns a new CompletionStage that, when this stage completes
```

Comment:
Seems a copy paste error - probably not a CompletionStage (here, in param and in return tags as well)

Reply:
Fixed thx